src/os_win32.c

```diff
@@ -334,7 +330,7 @@ read_console_input(
 
     if (s_dwMax == 0)
     {
-	if (!USE_WT && nLength == -1)
+	if (!CONPTY_STABLE && nLength == -1)
```

Comment:
not sure why this? workaround for buggy WT in win 10?   I think its no longer an issue in Win11, so I cant test this line on my pc at the moment.